app/views/test/components/student/tracks_list/show.html.haml

```diff
@@ -0,0 +1,11 @@
+%h1 Student
+%h2 Tracks list
+= form_tag nil, method: :get do
+  = label_tag :state
+  = select_tag :state, options_for_select(%w[Success Error Loading], params[:state])
+  = submit_tag "Submit", name: nil
+%br
+Error / Loading states are only seen when filtering
+%hr/
+
+= ViewComponents::Student::TracksList.new(@data, { endpoint: tracks_test_components_student_tracks_list_path, query: { state: params[:state] }, options: { retry: params[:state] == "Loading", initialStale: true } })
```

Comment:
I don't want to block this PR, but I think this can be improved lots. I've written up an issue: #139